smalldata/jira/full_testset.csv

```diff
@@ -0,0 +1,44 @@
+full,meat (gr), price (eur), restaurant, city, gender
```

Comment:
This file should not be used. We add files to our files differently, not by a commit.

Reply:
Same for shuffle_test_2.csv file.